ios/Podfile.lock

```diff
@@ -2868,8 +2868,8 @@ SPEC CHECKSUMS:
   web3.swift: 2263d1e12e121b2c42ffb63a5a7beb1acaf33959
   XMTP: 48d0c71ef732ac4d79c2942902a132bf71661029
   XMTPReactNative: 080300cc2cb53ffd117d2808c4d9922357ce1d34
-  Yoga: b05994d1933f507b0a28ceaa4fdb968dc18da178
+  Yoga: a9ef4f5c2cd79ad812110525ef61048be6a582a4
 
-PODFILE CHECKSUM: 8de80dcb8fc027dda19e4b9af8900634e49f9b40
+PODFILE CHECKSUM: e3b306b0081266b928275fbd16d3483e5a0ae250
 
-COCOAPODS: 1.16.2
+COCOAPODS: 1.15.2
```

Comment:
was cocoapods version downgrade intentional?

Reply:
nice catch :) fixed in 1f05d860736b851d9a62f8cc6de990a3a40cb76f